### PR TITLE
added parental "Machine" class with three more classes, looped throug…

### PR DESCRIPTION
**Pull request by vaniii1**
…h 'machines' array to do 'doWork()' and 'stopWork()' methods

### CODE REVIEW

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working");
+        return;
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
+        return;
+    }
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Bulldozer.java:13]:
```suggestion
public class Bulldozer extends Machine {
    @Override
    public void doWork() {
        System.out.println("Bulldozer is working");
    }
    
    @Override
    public void stopWork() {
        System.out.println("Bulldozer stopped working");
    }
```

Comment (kozhukhovsky) [reply src/main/java/core/basesyntax/Bulldozer.java:13]: Do it in other classes

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]
Java Cl checks failed. Before making a Pull Request (or a new commit to an already existing PR), you need to write the `mvn clean package` command in the terminal and make sure that all checks pass. After that, the mentor will be able to review the homework

**Comment by vaniii1** [comment]
дякую

ср, 19 апр. 2023 г. в 11:41, Anton Kozhukhovsky ***@***.***>:

**Comment by vaniii1** [comment]
отримав таку помилку

ср, 19 апр. 2023 г. в 11:53, Ivan Gromovoy ***@***.***>:

**Pull request closed without merging**